litgpt/eval/evaluate.py

```diff
@@ -69,6 +69,16 @@ def convert_and_evaluate(
 
     from lm_eval import evaluator
 
+    if tasks is None:
+        from lm_eval.tasks import TaskManager
+        taskm = TaskManager()
+
+        for key, value in taskm.task_index.items():
+            print(key)
+        raise ValueError(
+            "Tasks should be a comma-separated list of strings, for example, `'hellaswag,truthfulqa_mc2,mmlu'`"
+            "See the list of supported tasks above. To search for a specific task, use `litgpt evaluate | grep task_name`")
```

Comment:
This will produce a stack trace, and potentially just covering what you printed above. Wouldn't it be better if we did what download does?

https://github.com/Lightning-AI/litgpt/blob/e3e951148c14feb7f6dd782f7985350f9e2d86cc/litgpt/scripts/download.py#L41-L47

Reply:
I don't mind that. The reason why I used `raise ValueError` was to make it consistent with #1136, where I think someone suggested that approach. But let's maybe in both cases avoid raising an error and just print the list. I think this will look nicer to the user because otherwise a user might think they did something wrong.